app/facades/food_facade.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class FoodFacade
+  def self.search_foods(search_term)
+    foods = FoodService.search_foods(search_term)
+    foods[0..9].map do |data|
```

Comment:
Consider using the `pageSize` param provided by the API to limit the # of foods instead of Ruby (a bit cleaner)